### ISSUE TITLE
Add an 'About' popup

### ISSUE THREAD

**Issue by Infernio** [wrye-bash/wrye-bash#666]
*Literally only opening this right now because I want to snag #666 :P*

We should have an 'About' popup that informs users of:
 - Who made this program
 - WB's version
 - Versions of dependencies
 - WB's license (including a note about AGPL from PyMuPDF if used)
 - Third-party licenses (the whole reason I made that file machine-readable ;))
 - And a nice big logo

This is very simple, mostly just some GUI design. Still, I want it to show up on the changelog so an issue is needed.

**Comment by Arthmoor** [comment]
I recommend this be added to the start of a proper "Help" menu since that's where most "About" screens are linked from in other things. Definitely a good call though. The things we never think of until we do :P

**Comment by Infernio** [comment]
Yup, this will be the start of the 'Help' menubar item. I have some plans for interactive help in WB too, but that's far off.
